test/schema/home/home_page_artwork_modules.js

```diff
@@ -109,6 +116,7 @@ describe('HomePageArtworkModules', () => {
           'followed_galleries',
           'saved_works',
           'current_fairs',
+          'followed_artist',
           'related_artists',
```

Comment:
Awesome, so these two are always going to be next to each other, regardless of preferred order specified by the client?